src/vs/platform/windows/electron-main/windowImpl.ts

```diff
@@ -612,6 +614,40 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 		return this.marketplaceHeadersPromise;
 	}
 
+	private async updateChromiumSandboxEnablement(enable: boolean): Promise<void> {
```

Comment:
this duplicated from https://github.com/microsoft/vscode/blob/ac34ea3a6e3334c2bfb01f5c234863747e07f18c/src/vs/code/electron-main/app.ts#L1316, would it better to de-duplicate

Reply:
Yes that would be better. How about a reusable method in `src/vs/platform/environment/node/argv.ts`?